### PR TITLE
Add collapse_repeated to tensorflow frontend

### LINKED ISSUE #4396: collapse_repeated

### PR DESCRIPTION
**Pull request by Change-m**
Close #4396

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/nn.py [hunk 1]
```diff
@@ -123,3 +123,7 @@ def batch_normalization(x, mean, variance, offset, scale, variance_epsilon, name
     return x * ivy.astype(inv, x.dtype, copy=False) + ivy.astype(
         offset - mean * inv if offset is not None else -mean * inv, x.dtype
     )
+
+
+def collapse_repeated(labels, seq_length, name=None):
```
Comment (VedPatwardhan) [inline ivy/functional/frontends/tensorflow/nn.py:128]: I don't think there's a `collapse_repeated` function in Ivy's functional API, so either you'll have to implement the `tensorflow` `collapse_repeated` frontend as a composition of existing Ivy functions or extend Ivy's functional API by adding `ivy.collapse_repeated` in a separate PR referring to the [docs](https://lets-unify.ai/ivy/contributing/4_open_tasks.html#ivy-api-extensions)

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]

**Comment by VedPatwardhan** [comment]
I'll close this PR as discussed. Thanks 🙂

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**